src/gui/askexperimentalvirtualfilesfeaturemessagebox.cpp

```diff
@@ -0,0 +1,29 @@
+#include "askexperimentalvirtualfilesfeaturemessagebox.h"
+
+namespace OCC {
+
+AskExperimentalVirtualFilesFeatureMessageBox::AskExperimentalVirtualFilesFeatureMessageBox(QWidget *parent)
```

Comment:
Don't ask if vfs is non experimental.
![image](https://user-images.githubusercontent.com/200626/162916034-e3a31c71-c1c6-4010-aa7d-4208becb1ecf.png)